src/Indexing/Indexing.jsx

```diff
@@ -423,6 +427,23 @@ class Indexing extends React.Component {
       running: downloadManifestRunningPopupBlock,
     };
 
+    const downloadManifestSuccessButtons = [];
+    if (this.state.downloadManifestLogsLink) {
+      downloadManifestSuccessButtons.push({
+        caption: 'Download Logs',
+        icon: 'download',
+        value: this.state.downloadManifestLogsLink,
+        fn: () => this.downloadJobOutput(this.state.downloadManifestLogsLink),
+      });
+    }
+    downloadManifestSuccessButtons.push({
```

Comment:
should we check for `if (this.state.downloadManifestManifestLink)` here?

Reply:
I don't think so. The reason I added the check for logs is because Giang hasn't written code for the logs output for this generate manifest flow yet. That download manifest button already should not be displayed at all if the job was unsuccessful.